pom.xml

```diff
@@ -9,8 +9,8 @@
     <version>2021.07.29-SNAPSHOT</version>
 
     <properties>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Why do you rollback java version to 1.8?

Reply:
I was looking for the cause of the error, it was one of the tips in google